src/stdatamodels/jwst/datamodels/schemas/specwcs_miri_wfss.schema.yaml

```diff
@@ -0,0 +1,60 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/asdf/asdf-schema-1.0.0"
+id: "http://stsci.edu/schemas/jwst_datamodel/specwcs_miri_wfss.schema"
+title: SPECWCS_MIRI_WFSS reference file model
+allOf:
+- $ref: referencefile.schema
+- $ref: keyword_exptype.schema
+- $ref: subarray.schema
+- $ref: keyword_channel.schema
+- $ref: keyword_filter.schema
+- type: object
+  properties:
+    displ:
+        description: |
+          MIRI WFSS wavelength dispersion model, instance of astropy.modeling.Model
+        type: array
+        items:
+          type: object
+    dispx:
+        description: |
+          MIRI WFSS row dispersion model, instance of astropy.modeling.Model
+        type: array
+        items:
+            type: object
+    dispy:
+        description: |
+          MIRI WFSS column dispersion model, instance of astropy.modeling.Model
+        type: array
+        items:
+          type: array
+          items:
+            type: object
+    invdispl:
+        description: |
+          MIRI WFSS inverse wavelength dispersion model, instance of astropy.modeling.Model
+        type: array
+        items:
+          type: object
+    orders:
+        description: |
+          MIRI WFSS orders, matched to the array locations of the dispersion models
+        type: array
+        items:
+          type: number
+    meta:
+      type: object
+      properties:
+        input_units:
+          description: |
+            Input units for wavelength
+          anyOf:
+            - type: string
+            - $ref: http://stsci.edu/schemas/asdf/unit/unit-1.0.0
+        output_units:
+          description: |
+            Output units for wavelength
+          anyOf:
+            - type: string
+            - $ref: http://stsci.edu/schemas/asdf/unit/unit-1.0.0
```

Comment:
Should some of these be made `required`?

Reply:
Honestly I do not know. None of the other grism models have required -but maybe they should be. What makes something 'required' in a schema. We need them all  - so are they all required ?

Reply:
I don't think it's a good idea to touch the other instruments in this PR. Also I would leave the `input_units` and `output_units` out of `required` because they are not essential for the transform, only informational.

Reply:
@nden By the way as we look at better way to do things in the future. In the datamodels there is hardly any required values in the datamodels and I would think that many of the values would be required.  We might want to take another look at that issue.